<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CityDbExplorerDockWidgetBase</class>
 <widget class="QDockWidget" name="CityDbExplorerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DCityDB Explorer</string>
  </property>
    <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Databases</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="databaseConnections"/>
      </item>
      <item>
       <widget class="QPushButton" name="connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Max features</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxFeatures">
          <property name="text">
           <string>1000</string>
          </property>
          <property name="placeholderText">
           <string>1000</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="dbVersion">
        <property name="text">
         <string>Please select a connection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="loadBuilding">
        <property name="text">
         <string>Add buildings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editAttribute">
        <property name="text">
         <string>Edit Attribute</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
